gnosis/dex-services
# Enable rust e2e test for rinkeby

The fact that https://github.com/gnosis/dex-services/pull/420 made it into master is mainly caused by the fact that our rinkeby e2e tests were not correctly functioning.

We were matching case sensitively on "ERROR" which in a recent logging refactoring changed into "ERRO".

Instead of fixing up the legacy e2e test (which is turning out to have other problems such as not waiting for confirmations etc.) I'm replacing it with the new rust based e2e test.

I tried to see if there is a more generic way to filter errors (e.g. by looking at STDERR but that doesn't seem to be possible with docker-compose logs).


### Test Plan

- rebase this PR on top of 2a3385b6e3d9d4d713e456efe0385ac0c5413aeb
- run `docker-compose -f docker-compose.yml -f docker-compose.rin
keby.yml up stablex`
- run `cargo test -p e2e rinkeby -- --nocapture`

See e2e test failing.
